pymeasure/instruments/jobinyvon/jobinyvon270m.py

```diff
@@ -0,0 +1,360 @@
+        code = self._get_code(ans)
+        if code != 'o':
+            raise IOError(f'Wrong return code from driver, received {code}')
+
+    def get_entry_slit_microns(self):
+        """
+        Reading of the ABSOLUTE position of the entrance slit in micrometres.
+        """
+        return self.entrysteps / self._slit_steps_micron
+
+    def move_entry_slit_microns(self, microns: float):
+        """
+        ABSOLUTE positioning of the entry slit in micrometers.
+        """
+        pos_steps = microns * self._slit_steps_micron
+        pos_steps = int(pos_steps)
+        self.move_entry_slit_steps(pos_steps)
+
+    def move_exit_slit_steps(self, nsteps: int):
+        """
+        ABSOLUTE positioning of the exit slit motor in number of steps.
+        """
+        ans = self.write_read(f'k0,2,{nsteps - self.exitsteps}\r'.encode(), nread=1, timeout=10000)
+        time.sleep(0.5)
+        code = self._get_code(ans)
+        if code != 'o':
+            raise IOError(f'Wrong return code from driver, received {code}')
+
+    def get_exit_slit_microns(self):
+        """
+        Reading of the ABSOLUTE position of the exit slit in micrometres.
+        """
+        return self.exitsteps / self._slit_steps_micron
+
+    def move_exit_slit_microns(self, microns: float):
+        """
+        ABSOLUTE positioning of the exit slit in micrometres.
+        """
+        pos_steps = microns * self._slit_steps_micron
+        pos_steps = int(pos_steps)
+        self.move_exit_slit_steps(pos_steps)
+
+    def check_get_errors(self, error):
+        print(error)
+        pass
+
+    def motor_stop(self):
+        """
+        Stopping of all the motors when this function is called.
+        """
+        ans = self.write_read(b'L')
+        code = self._get_code(ans)
+        if code != 'o':
+            raise IOError(f'Wrong return code from driver, received {code}')
+
+    def motor_busy_check(self):
+        """
+        This function checks if at least one of the motors is busy.
+        """
+        ans = self.write_read(b'E\r').decode()
+        if ans == "oz":
+            "We return False if the motor is not busy."
+            return False
+        if ans == "oq":
+            "We return True if the motor is busy."
+            return True
+
+    def motor_available(self):
+        """
+        This function verifies that the grating motor or the slit motors are free.
+        """
+        while self.motor_busy_check():
+            pass
+        time.sleep(1)
+        return True
+
+
+if __name__ == '__main__':
+    spectro = JY270M('COM1',
+                     baud_rate=9600,
+                     timeout=300,
+                     parity=Parity.none,
+                     data_bits=8,
+                     stop_bits=StopBits.one,
+                     flow_control=ControlFlow.dtr_dsr,
+                     write_termination='',
+                     read_termination='',
+                     includeSCPI=False)
```

Comment:
Please remove these lines.

Reply:
This `if name == main` section is still present.
However, it is useless, as the device will have a different port number (actually it is outdated and won't work).
Additionally, we do not use this kind of initialisator in the driver files.